spec/index.html

```diff
@@ -791,7 +791,7 @@ <h3>Literals</h3>
     <p>Literals are used for values such as strings, numbers, and dates.</p>
 
     <p>A <dfn data-local-lt="RDF literal">literal</dfn> in an <a>RDF graph</a> consists of
```

Comment:
Mentioning "an RDF graph" here is incorrect. The notion of a literal should be defined independent of any RDF graph.
```suggestion
    <p>A <dfn data-local-lt="RDF literal">literal</dfn> consists of
```

Reply:
agreed. I actually found it strange when re-reading it, but I was focusing on the changes suggested during review :)